ChangeLog.d/fix-rsa-performance-regression.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix performance regression when using short RSA public keys.
```

Comment:
Minor nits: (1) we don't totally fix the performance regression, but we reduce the impact on public-key operations of the constant-time changes in 3.6; (2) I'm pretty sure there was an issue raised for this, so should we mention the issue number?

Reply:
(1) The rest of the performance regression is the price of better security. The phrasing you suggest would imply that we treat that as a regression and it will be fixed, which is not the case.
(2) I was thinking about it, but I didn't add as I didn't want to auto-close the issue. But I guess we can always reopen it.